pyproject.toml

```diff
@@ -267,6 +267,10 @@ env = [
 filterwarnings = [
     "error",
     "always::DeprecationWarning",
+    # This warning is issued in two contexts, and we ignore both:
+    # Python 3.10, 3.11: sys.monitoring isn't available in this version, using default core (no-sysmon)
+    # Python 3.12, 3.13: sys.monitoring can't measure branches in this version, using default core (no-sysmon)
+    "always::coverage.exceptions.CoverageWarning",
```

Comment:
We can probably completely ignore the first warning (for Py 3.10 and 3.11) because that’s expected behavior (see [Warnings Filter](https://docs.python.org/3/library/warnings.html#warning-filter). But we should not see the second one warning… 🤔